PowerEditor/src/WinControls/Preference/preferenceDlg.cpp

```diff
@@ -1251,6 +1254,13 @@ intptr_t CALLBACK TabbarSubDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM
 			if (hideTabbar)
 				::SendMessage(_hSelf, WM_COMMAND, IDC_CHECK_TAB_HIDE, 0);
 
+			::SetDlgItemInt(_hSelf, IDC_EDIT_TABCOMPACTLABELLEN, nppParam.getNbTabCompactLabelLen(), FALSE);
+
+			NativeLangSpeaker* pNativeSpeaker = nppParam.getNativeLangSpeaker();
+			wstring tabCompactLabelLenTip = pNativeSpeaker->getLocalizedStrFromID("tabbar-tabcompactlabellen-tip",
+				L"This is to limit the visible length of the tab name and thus the tab size. The allowed range is 0 - 257 characters, with 0 meaning that the compacting is disabled.");
+			_tabCompactLabelLenTip = CreateToolTip(IDC_TABCOMPACTLABELLEN_TIP_STATIC, _hSelf, _hInst, const_cast<PTSTR>(tabCompactLabelLenTip.c_str()), pNativeSpeaker->isRTL());

```

Comment:
Notepad++ should remove all "T" string types like `PTSTR`. 
`const_cast<PTSTR>(tabCompactLabelLenTip.c_str())` can be replaced with `tabCompactLabelLenTip.data()` I think